src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/model/ccd/ExceptionRecord.java

```diff
@@ -43,6 +43,10 @@
     @JsonProperty("awaitingPaymentDCNProcessing")
     public final String awaitingPaymentDcnProcessing;
 
+    // Yes/No field indicating if the exception record contains payments
+    @JsonProperty("containsPayments")
+    public final String containsPayments;
```

Comment:
can this be boolean?

Reply:
It can be, but we have to send Yes/No value in the CCD request

Reply:
ok

Reply:
Actually, it can't be boolean. this is the JSON property type and it should be `string`.
I was talking about the construction parameter earlier.